packages/arb-ts/scripts/deployStandard.ts

```diff
@@ -0,0 +1,131 @@
+import { instantiateBridge } from './instantiate_bridge'
+import { ERC20__factory } from '../src/lib/abi/factories/ERC20__factory'
+import dotenv from 'dotenv'
+import args from './getCLargs'
+import { constants, BigNumber, utils } from 'ethers'
+
+dotenv.config()
+
+const privKey = process.env.PRIVKEY as string
+if (!privKey) {
+  throw new Error('Set PRIVKEY env variable')
+}
+
+if (!args.l1_token_address) {
+  throw new Error(
+    'Include l1_token_address arg (--l1_token_address 0xmytokenaddress)'
+  )
+}
+if (!args.amount) {
+  throw new Error('Include amount arg (--amount 123)')
+}
+
+const { l1_token_address: l1TokenAddress, amount } = args
+const amountBigNum = BigNumber.from(amount)
+
+const main = async () => {
+  const { bridge } = await instantiateBridge(privKey, privKey)
+  const walletAddress = await bridge.l1Bridge.getWalletAddress()
+
+  const l1Token = ERC20__factory.connect(l1TokenAddress, bridge.l1Provider)
+  // TODO: check blacklist
+
+  /* Looks like an L1 token: */
+  try {
+    await l1Token.symbol()
+    await l1Token.name()
+    await l1Token.decimals()
+    await l1Token.balanceOf(walletAddress)
```

Comment:
we can multicall these

Reply:
i think that if you use the L1 bridge's "get token data" it should work
https://github.com/OffchainLabs/arbitrum/blob/2c2cd9872e959c77be45a72af075c67379658cad/packages/arb-ts/src/lib/l1Bridge.ts#L133